csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+For example, if we had 16 sticky notes under “waking up unrested” and only 5 under “unsure about sleep requirements” and “staying up too late” then we would confidently consider “waking up unrested” to be a high priority concern for our users.
+
+{panel type="teacher-note"}
+
+# Affinity Diagram Tips
+
+This is a very small affinity diagram.
+Your students should have a lot of data to work with from their surveys and therefore should have at least 20 sticky notes that they can sort into 4 or 5 strong themes.
+
+{panel end}
+
+### Step 4: Describe your user.
+
+**UX Methods: Empathy Map, User Persona and User Stories**
+
+Now that we’ve established who our users are and what they want, it would be useful to create a *User Persona*.
+A *User Persona* is a character that is representative of your target user.
+To help us with creating a user persona, we will use an *Empathy Map* to give us a structure to work with.
+An *Empathy Map* defines four aspects of the persona: how they Think and Feel and what they Do and Say, and is drawn in quadrants.
+We’ve created a very basic one below to help us develop a persona for the target user of our sleep habits app:
+
+|                                                                                                                                          	| **User**                                                                                                                                                                                                                    	|
+|------------------------------------------------------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| **Says**<br>Things the user says out loud in an interview or wrote in their survey.<br><br>*I am so tired.<br>I think I stayed up too late last night.* 	| **Thinks**<br>What do they think about? What are their needs, goals and motivations?<br><br>*I wish I could get more sleep.<br>Maybe I should go to bed earlier.<br>How many hours of sleep do I actually need?<br>Why am I always tired?* 	|
+| **Does**<br>What actions does the user do?<br><br>*Always hits “snooze” on the alarm.<br>Drinks coffee first thing in the morning.*                     	| **Feels**<br>Consider the tone they use in an interview and/or survey answers.<br><br>*Tired.<br>Annoyed.<br>Frustrated.*                                                                                                                	|
+
+Using our Empathy Map we are able to derive the target users needs and from here we created a persona representative of our target user.
+Meet Bevan:
+
+**Bevan, 22, Accounting Student at University**
+
+{image file-path="img/chapters/bevan.jpg" alt="Photo of Bevan for persona."}
```

Comment:
This photo stands out (not in the right way) compared to the rest of the section (and the whole website). An cartoon/outline fits our design better.

Maybe it's just his eyes... 👀

Reply:
I don't think we need a photo

Reply:
Maybe we don't need a photo/cartoon at all? We don't want people to form an opinion and be subjective based on the photo :) (Just my opinion though).

Reply:
My HCI lecturer was very insistent that an image was required for this sort of study to be successful; something about helping with empathy and giving a face to data, even hypothetical

Reply:
Interesting @eAlasdair, I can see the points for both sides. We do want to connect with a human person.

I found this [article](https://www.cxpartners.co.uk/our-thinking/dont-use-photos-in-your-personas/) which raised some good points (and I'm sure there is articles both ways):

> Demographics don’t help me design. Knowing gender, age and the size of someone’s house doesn’t tell me how to shape a service. I prefer personas that strip out demographic information and focus on what they do, their goals, behaviours, and how we must support them.

I do think there is a place for when demographics are useful, but in our case (as they also experienced) cartoons would get the goal across. I put more comments about this in the inclusiveness discussion below.

Reply:
Can we please reduce the size of this image? (Even though #992 will be addressed ASAP)